oeps/oep-0044-bp-changelogs.rst

```diff
@@ -0,0 +1,212 @@
+##################
+OEP-44: Changelogs
+##################
+
+.. list-table::
+
+   * - OEP
+     - :doc:`OEP-0044 </oeps/oep-0044>`
+   * - Title
+     - Changelogs
+   * - Last Modified
+     - 2020-04-30
+   * - Authors
+     - Robert Raposa <rraposa@edx.org>
+   * - Arbiter
+     - ???
+   * - Status
+     - Draft
+   * - Type
+     - Best Practice
+   * - Created
+     - 2020-03-19
+   * - Review Period
+     - ???
+
+.. contents::
+   :local:
+   :depth: 3
+
+Summary
+=======
+
+.. note::
+
+    This OEP is on hold at this time while some of these recommendations are first trialed in some repositories.
+
+This OEP contains recommendations rather than requirements for changelogs.
+
+* The recommendation is for each repository to have a CHANGELOG.rst.
+* To learn about maintaining a changelog, see `Formatting Changelogs`_, `Example Changelog`_, and `Commit Message vs Changelog Entry`_.
+* For more general information, see the `What and Why of Changelogs`_.
+
+Context
+=======
+
+Changelogs are handled differently in many Open edX repositories, and this OEP provides guidance if you are wondering which method to choose.
+
+What and Why of Changelogs
+==========================
+
+A useful reference is `keepachangelog.com`_, although this OEP is less strict in its recommendations.
+
+The following questions and answers are copied from `keepachangelog.com`_
+
+* **What is a changelog?**: A changelog is a file which contains a curated, chronologically ordered list of notable changes for each version of a project.
+* **Why keep a changelog?**: To make it easier for users and contributors to see precisely what notable changes have been made between each release (or version) of the project.
+* **Who needs a changelog?**: People do. Whether consumers or developers, the end users of software are human beings who care about what's in the software. When the software changes, people want to know why and how.
+
+Recommendations
+===============
+
+All Open edX repositories should contain a CHANGELOG.rst.
+
+The recommendations for formatting changelogs have been adapted from `keepachangelog.com`_.
+
+Formatting Changelogs
```

Comment:
Consider relying more heavily on keepachangelog.com default recommendations and simplifying, like in https://github.com/edx/edx-drf-extensions/commit/f5f7b50dac1c8116c8dccc1d9809a9ee568d14e7